### ISSUE TITLE
RELEASE READINESS | User Story 19: Get a cloud machine with complete control in Mumbai

### ISSUE THREAD

**Issue by bevanamit** [theheapdump/kite-webapp#20]
Would require 2CPU and at least 2 GB RAM with 20 GB working space

**Comment by theheapdump** [comment]
2 CPU 
8 GB RAM
50GB HDD

we might have to run docker also - so good cpu and ram. needed 
storage for DB !!

**Comment by bevanamit** [comment]
I was thinking for inital run do we need so much?.. In  case we can run
with low we can opt for it... In case we need more slowly we can
increase... Basically trying to cut cost

**Comment by theheapdump** [comment]
yes, we can start slow - but before release, we need load testing so that if needed we can upgrade. 
but 50GB we need.

RAM, we can go slow

**Comment by bevanamit** [comment]
Yes

**Comment by theheapdump** [comment]
@bevanamit  what about this one ??

**Comment by bevanamit** [comment]
Will keep this open for now. As this is for launch
